tests/api/ils/test_patrons.py

```diff
@@ -0,0 +1,83 @@
+import json
+
+from flask import url_for
+from tests.helpers import user_login
+
+
+def _patron_loans_request(client, json_headers, document_pid):
+    """Perform a patron loans request."""
+    params = []
+    params.append("document_pid={}".format(document_pid))
+    response = client.get(
+        "{}?{}".format(
+            url_for("invenio_app_ils_patrons.get_user_information"),
+            "&".join(params),
+        ),
+        headers=json_headers,
+    )
+    return json.loads(response.data.decode("utf-8"))
+
+
+def _assert_patron_loans_information(client, json_headers, document_pid,
+                                     expect):
+    """Assert patron loan information."""
+    resp = _patron_loans_request(client, json_headers, document_pid)
+    assert resp == expect
+
+
+def _assert_has_no_authorization(client, json_headers, document_pid, expect):
+    """Assert user is not logged in."""
+    resp = _patron_loans_request(client, json_headers, document_pid)
+    if resp['status'] == 401:
+        assert resp['status'] == expect['status']
+
+
+def test_patron_loans_information(
+    client, json_headers, testdata_most_loaned, users
+):
+    """Test patron loans information API endpoint."""
+    _assert_has_no_authorization(
```

Comment:
```suggestion
    _assert_error_when_not_authenticated(
```
I am being picky here, sorry! This is about authentication, and not authorization. :)